### ISSUE TITLE
Microsoft Edge fails to launch properly when using capemon 8aa7e30

### ISSUE THREAD

**Issue by seanthegeek** [kevoreilly/CAPEv2#1587]
# Expected Behavior

Microsoft Edge should launch correctly when a URL analysis is requested.

# Current Behavior

Microsoft Edge does not launch when `capemon` from commit 8aa7e30 is used.

# Failure Information (for bugs)

Reverting to `capemon` from commit e1caf4f fixes the issue.

## Steps to Reproduce

1. Run a URL analysis

## Context

| Question         | Answer
|------------------|--------------------
| Git commit       | 8aa7e30
| OS version       | Windows 10 

## Failure Logs

[27_analysis.log](https://github.com/kevoreilly/CAPEv2/files/11682135/27_analysis.log)
[27_report.json.txt](https://github.com/kevoreilly/CAPEv2/files/11682137/27_report.json.txt)

**Comment by CarsonHrusovsky** [comment]
I have moved to firefox and have not found any issues yet with it

**Comment by seanthegeek** [comment]
I've just confirmed that this occurs in both Edge 94 and 114

**Comment by kevoreilly** [comment]
Thanks for the report - fixed with the commit mentioned above

**Comment by seanthegeek** [comment]
Well, this is interesting. After I upgraded to this release of `capemon` I started having issues with crashes and black screens either shortly after the URL analysis starts, or near the end of a 200 second analysis using the version of Edge that comes with the Windows 10 21H2 install ISO (92.0.902.67). I think it was something to do with the injection into `services.exe` that you talked about.

https://github.com/kevoreilly/CAPEv2/issues/1588#issuecomment-1589288367

> There are two likely candidates when it comes to bsod - as capemon is a usermode dll it has limited opportunities to cause full system crashes. The two most likely possibilities are after injection into services.exe (for certain types of detonation) or injection into lsass.exe as part of the tldump feature implemented in an auxiliary module.

I have several update-related services for Microsoft Edge disabled, because the update GPO policies are only monitored on domain-joined systems.

- Microsoft Edge Elevation Service (MicrosoftEdgeElevationService)
- Microsoft Edge Update Service (edgeupdate)
- Microsoft Edge Update Service (edgeupdatem)

Edge really hates it when you do that, to the point where on newer builds of Edge, the entire system will crash if you try to use the browser with those services disabled, even without CAPE running at all.
